data/version.json

```diff
@@ -1 +1,5 @@
-{ "read_me": "CURRENT_VERSION IS NO LONGER UPDATED. USE VERION", "current_version": 999999, "version": "4.2.1" }
+{
+  "read_me": "CURRENT_VERSION IS NO LONGER UPDATED. USE VERION",
+  "current_version": 999999,
+  "version": "4.2.1"
+}
```

Comment:
Newline, please. And please keep spacing (4 spaces) consistent with the rest of the project